decidim-core/lib/decidim/api/types/attachment_type.rb

```diff
@@ -4,12 +4,18 @@ module Decidim
   module Core
     class AttachmentType < Decidim::Api::Types::BaseObject
       description "A file attachment"
+      implements Decidim::Core::TimestampsInterface
 
+      field :content_type, GraphQL::Types::String, "The content type of this attachment", null: true
```

Comment:
What is the difference between `content_type` and `type` (line 16)?

Reply:
The content type would be matched to "image/jpeg", but the "type" would be matched to either "Jpeg" or "link" when we have link attachments.